governance/default_test.go

```diff
@@ -803,6 +803,23 @@ func TestGovernance_HandleGovernanceVote_None_mode(t *testing.T) {
 	}
 	gov.voteMap.Clear()
 
+	//////////////////////////////////////////////////////////////////////////////////////////////////////////
+	// Test adding an existing validator
```

Comment:
How about adding remove not exist validator test?

Reply:
Added. PTAL